crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md

```diff
@@ -0,0 +1,53 @@
+---
+id: integration_securitycopilot
+title: Security Copilot Plugin
+---
+
+CrowdSec Intelligence Plugin for Microsoft Security Copilot allows you to get advanced insights on a malicious IP activity.  
+As part of the core plugins of Security copilot its setup and usage are very straight forward.
+
+This documentation will lead you through an easy setup and lead you through some example usage and prompts.
+
+## Configure the plugin
+
+### Prerequisite: retrieve your API Key
+The plugin is using our CTI API to provide information on over 70M attackers recently reported by CrowdSec's network.  
+You can create a trial key or retrieve your existing keys in the [console](https://app.crowdsec.net/) in the "Settings" > "CTI API Keys" section.  
+If you need more details check out the [CTI API Key - getting started section](/cti_api/getting_started)
```

Comment:
Can you make a ref to the file itself `(/cti_api/getting_started.mdx)` as the compiler will auto link the actual route

Reply:
As using it without leads to a 404 https://pr-566.d1to60jd2gb6y6.amplifyapp.com/cti_api/getting_started

Reply:
```suggestion
If you need more details check out the [CTI API Key - getting started section](/cti_api/getting_started.mdx)
```